Snippets/Snippets_6/Transactions.cs

```diff
@@ -32,17 +50,17 @@ public void TransportTransactionsWithScope()
         {
             #region TransactionsWrapHandlersExecutionInATransactionScope
             BusConfiguration busConfiguration = new BusConfiguration();
-            busConfiguration.Transactions()
-                .WrapHandlersExecutionInATransactionScope();
+            busConfiguration.UnitOfWork()
+                .WrapHandlersInATransactionScope();
             #endregion
         }
 
-        public void CustomTransactionTimeout()
+        public void UnitOfWorkWrapHandlersInATransactionScope()
         {
-            #region CustomTransactionTimeout
+            #region UnitOfWorkWrapHandlersInATransactionScope
```

Comment:
The custom transaction timeout snippet exists in all versions: 3.x, 4.x, 5.x, and 6-pre. For 6-pre it is defined here:

https://github.com/Particular/docs.particular.net/pull/1026/files#diff-e43503f5ec8194d91e9e461683ad48a4R77

Does this make sense?